builder/decorators.py

```diff
@@ -32,7 +32,9 @@ def require_permission(view_fn):
 
     @wraps(view_fn)
     def wrapped_view(request, draft, **kwargs):
-        if request.user == draft.author:
+        if request.user == draft.author or draft.codelist.can_be_edited_by(
+            request.user
+        ):
```

Comment:
Ah, I had thought that this was an error I'd introduced when I did the work on authors, but it was just a name change.  I'll remove it, but I think we should have some sort of indication on the builder page that you don't have permission to edit it.  It took me a while to realise that the +/- buttons weren't working because I wasn't the draft's author, not because of a frontend bug.  It wasn't until I noticed that the search field wasn't there that I figured it out.  I think it's quite likely that someone might make a draft and then send the link to someone else to look at, which will work for them, but won't be editable